airbyte-cdk/bulk/toolkits/load-db/src/main/kotlin/io/airbyte/cdk/load/orchestration/DestinationNames.kt

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2025 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.load.orchestration
+
+import io.airbyte.cdk.load.command.DestinationStream
+
```

Comment:
this file supersedes the StreamConfig / StreamId / ColumnId stuff that the old CDK used. Mostly b/c the bulk CDK's DestinationStream is a clear replacement for StreamConfig.

It's a bit less ergonomic, in that instead of passing a single StreamConfig around, we have to pass around tuples of DestinationStream x TableName(s) x ColumnNameMapping. But I kind of like the explicitness, that every method now only accepts exactly what it needs.